addons/product_expiry/tests/test_stock_lot.py

```diff
@@ -685,3 +685,77 @@ def test_compute_expiration_date_from_scheduled_date(self):
         delivery.action_confirm()
 
         self.assertAlmostEqual(delivery.move_line_ids[0].expiration_date, expiration_date, delta=delta)
+
+    def test_assign_lot_expiry_alert_to_default_user(self):
+        """ Test lot expiry alert is assigned to the default user of the activity type """
+
+        # User A will be the Default User on the activity type
+        user_a, user_b = self.env['res.users'].create([
```

Comment:
would be great to rename to `default_user` and `responsible_user` or something like that.